specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/applicationGateway.json

```diff
@@ -451,6 +451,96 @@
         },
         "x-ms-long-running-operation": true
       }
+    },
+	"/subscriptions/{subscriptionId}/providers/Microsoft.Network/applicationGatewayAvailableServerVariables": {
+      "get": {
+        "tags": [
+          "ApplicationGateways"
+        ],
+        "operationId": "ApplicationGateways_ListAvailableServerVariables",
+        "x-ms-examples": {
+          "Get Available Server Variables": {
+            "$ref": "./examples/ApplicationGatewayAvailableServerVariablesGet.json"
+          }
+        },
+        "description": "Lists all available server variables.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
there should be a "default" response with error schema

Reply:
We have only one response for this API. So it is default only. Please let me know if i'm missing anything.

Reply:
Will this API never return an error? The "default" response is a catch-all for non-success responses:
```         "default": {
	            "description": "Error response describing why the operation failed.",
	            "schema": {
	              "$ref": "#/definitions/CloudError"
	            }
	          }
```

Reply:
Added the "default" error message.